utils/get_info.py

```diff
@@ -121,14 +125,47 @@ def volume_counter(root, subject, ses=None, tr=1.49):
 
     return ses_runs
 
+def get_acq_channels(root, acq_file):
+    """
+    Get the names of the channels in the acq file 
+
+    Parameters:
+    ------------
+    root : path
+        directory containing the biopac data
+    acq_file : string
+        name of the acqknowledge file
+    
+    Returns:
+    --------
+    ch_name : list
+        list of the channel names in the same order as they are in the acq file
+    """
+    read_acq = bioread.read_file(os.path.join(root, acq_file))
+    ch_name = []
+    for ch in read_acq.channel_headers:
+        if 'PPG' in ch.name:
+            ch_name.append('PPG')
+        elif 'ECG' in ch.name:
+            ch_name.append('ECG')
+        elif 'HLT' in ch.name:
```

Comment:
We might want to check channels from other tasks, as we've changed the setup for the RSP. We used to have a custom channel for both TTL and RSP

Reply:
Maybe we could make the function more general by adding arguments for the different type if signals:
`get_acq_channels(root, acq_file, ppg='PPG', ecg='ECG', respiration='DA100C', trigger='HLT )`. It would be easier to change if needed